src/Screens/ChatScreen.tsx

```diff
@@ -81,15 +92,92 @@ export default function ChatScreen({route, navigation}: any) {
       });
   };
 
+  const isFocused = useIsFocused();
+
+  const fetchSingleDialog = () => {
+    setLoading(true);
+    console.log('dialogId', dialog?.id);
+    QB.chat
+      .getDialogs(dialog?.id)
+      .then((dialog: any) => {
+        time = dialog?.dialogs?.reduce(
+          (max: number, obj: any) =>
+            obj.customData.time > max ? obj.customData.time : max,
+          -Infinity,
+        );
+        cacluatetime();
+      })
+      .catch(error => {
+        console.error('Error retrieving dialog:', error);
+      });
+  };
+
+  useEffect(() => {
+    fetchSingleDialog();
+    return () => {
+      if (intervalRef.current) {
+        clearInterval(intervalRef.current);
+        time = 0;
+      }
+    };
+  }, []);
+
+  const cacluatetime = () => {
+    if (isFocused) {
+      intervalRef.current = setInterval(() => {
+        if (time === 600 || time > 600) {
```

Comment:
You can optimize the condition time === 600 || time > 600 by simplifying it to just time >= 600